drivers/hmis/spec/models/hmis/client_alert_spec.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'rails_helper'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.describe Hmis::ClientAlert, type: :model do
+  include_context 'hmis base setup'
+  include_context 'hmis service setup'
+
+  before(:all) do
+    cleanup_test_environment
+  end
+  after(:all) do
+    cleanup_test_environment
+  end
+
+  include_context 'hmis base setup'
+
+  describe 'basic client alert tests' do
+    let!(:c1) { create :hmis_hud_client, data_source: ds1 }
+    let!(:a1) { create :hmis_client_alert, created_by: hmis_user }
+
+    it 'should be able to create a client alert' do
+      expect(a1.created_by.class).to eq(Hmis::User)
+    end
+  end
```

Comment:
I don't think this is testing the permission logic, just that the relation `created_by` is working.  To test the permissions you'll need to go through the graphQL endpoint with a submission that creates the alert.

Reply:
she is doing the graphql endpoints in a separate pr, just the setup here

Reply:
That's fine, but checking the class of the created_by relation isn't actually checking `it 'should be able to create a client alert'`  Only that the save worked successfully and the relation is correctly setup.

Reply:
From talking with Gig earlier my understanding is that the Open Path norm would be not to add any tests testing the functionality I added in this PR, is that right? Should I just delete this test file in this PR?

Reply:
No, don't delete it!  We may not be great at adding tests for everything, but we always appreciate when someone takes the time to add them.  I was more commenting that the 'it' line doesn't reflect what the test is doing.  perhaps the following would be a more accurate description:
```suggestion
  describe 'basic client alert tests' do
    let!(:c1) { create :hmis_hud_client, data_source: ds1 }
    let!(:a1) { create :hmis_client_alert, created_by: hmis_user }

    it 'client alert is saved successfully and connected to the saving user' do
      expect(a1.created_by.class).to eq(Hmis::User)
    end
  end
```

Reply:
Oh perfect, thank you for clarifying!!